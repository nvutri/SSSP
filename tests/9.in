c random graph n=32 m=128 lo=0 hi=262144 seed=1953044356
p sp 32 128
a 14 4 208693
a 4 13 146935
a 13 32 123605
a 32 19 148463
a 19 22 204097
a 22 8 89428
a 8 2 45722
a 2 5 74547
a 5 25 245648
a 25 6 93937
a 6 21 131378
a 21 10 240881
a 10 30 179253
a 30 1 129863
a 1 28 217079
a 28 20 223709
a 20 3 173635
a 3 24 21482
a 24 17 162287
a 17 23 112939
a 23 11 154451
a 11 7 128864
a 7 9 1939
a 9 16 28802
a 16 31 128117
a 31 15 108833
a 15 27 98763
a 27 26 27126
a 26 18 229372
a 18 29 91863
a 29 12 158913
a 12 14 43360
a 2 29 21349
a 5 15 28464
a 6 26 151627
a 28 30 228476
a 1 15 1882
a 20 3 160968
a 3 24 94438
a 31 26 232379
a 14 3 194294
a 25 2 206357
a 28 18 252964
a 11 30 248032
a 20 2 185675
a 4 9 114900
a 24 3 240041
a 29 30 88824
a 12 32 104064
a 6 5 178993
a 3 23 252306
a 20 8 160445
a 8 28 208548
a 27 31 130105
a 1 30 114360
a 19 9 54145
a 5 31 75186
a 16 28 106492
a 25 9 46595
a 27 18 44241
a 20 3 86688
a 17 24 160956
a 25 8 235147
a 27 30 138908
a 10 18 156976
a 20 5 94979
a 21 3 204123
a 14 13 89714
a 15 27 36400
a 9 15 201197
a 25 12 22432
a 24 28 61694
a 21 5 163999
a 23 11 80235
a 28 9 125933
a 22 24 226950
a 30 8 203152
a 6 15 87254
a 18 2 237582
a 8 27 49310
a 31 23 56087
a 27 14 177085
a 25 2 192014
a 25 6 113682
a 6 4 145047
a 31 24 198072
a 16 18 260434
a 22 23 153311
a 24 12 90450
a 28 9 229099
a 13 6 216574
a 9 17 108710
a 15 31 215095
a 1 17 179479
a 2 23 132453
a 15 9 48745
a 29 15 104764
a 13 32 224938
a 27 17 20198
a 5 19 194154
a 6 24 107297
a 24 26 254454
a 12 23 116620
a 21 32 260421
a 16 19 231942
a 17 10 7
a 25 18 95256
a 1 16 196875
a 21 20 99302
a 30 8 33609
a 23 13 109796
a 31 20 129970
a 3 11 155715
a 6 20 32779
a 31 9 5580
a 28 5 241387
a 15 31 24207
a 20 9 14410
a 10 12 208203
a 25 7 243797
a 17 18 6632
a 2 6 194949
a 31 4 17731
a 29 17 72274
a 1 7 90145
a 25 14 20518
a 5 32 59906
a 12 2 257370
