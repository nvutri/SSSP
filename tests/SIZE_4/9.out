1 0
2 193691
3 242747
4 43820
5 251038
6 388640
7 90145
8 147969
9 31669
10 58487
11 162411
12 266690
13 190755
14 215367
15 1882
16 60471
17 58480
18 65112
19 292413
20 156059
21 520018
22 496510
23 82176
24 219436
25 496686
26 65408
27 38282
28 166963
29 156975
30 114360
31 26089
32 310944
